### PR TITLE
Format multiline locals by indenting all lines uniformly #92

### PR DESCRIPTION
**Pull request by leifwalsh**
Still needs tests, and probably needs to update some other tests' expected strings.

### CODE REVIEW

**Comment by leifwalsh** [comment]
The example from #92 looks like this with this patch:

```
Source (/home/leif/git/marbles-demo-bikeshare/bikeshare/test_bikeshare.py):
     49 long_trips = _data[_data['tripduration'] > pd.Timedelta('24h')]
 >   50 self.assertDataFrameEmpty(long_trips)
     51 
Locals:
	long_trips =
		            tripduration               starttime                stoptime
		1495     1 days 13:33:16 2015-01-08 01:06:37.000 2015-01-09 14:39:54.000
		4064     1 days 21:25:08 2015-01-17 13:55:59.000 2015-01-19 11:21:08.000
		7168     2 days 16:31:59 2015-01-26 17:14:12.000 2015-01-29 09:46:11.000
		722      1 days 19:32:50 2015-02-06 15:31:02.000 2015-02-08 11:03:53.000
		1388     1 days 02:19:46 2015-02-13 08:04:02.000 2015-02-14 10:23:49.000
		5621     6 days 16:32:20 2015-03-17 16:26:54.000 2015-03-24 08:59:15.000
		6453   191 days 14:29:48 2015-03-20 02:24:09.000 2015-09-27 16:53:58.000
```

File: marbles/core/marbles/core/marbles.py [hunk 1]
```diff
@@ -328,9 +328,19 @@ def formattedMsg(self):  # mimic unittest's name for standardMsg
             standardMsg=self.standardMsg, assert_stmt=self.assert_stmt,
             note=self.note, locals=local_string, filename=self.filename)
 
+    @classmethod
+    def _format_one_local(cls, name, value):
```
Comment (thejunglejane) [inline marbles/core/marbles/core/marbles.py]: Why `_format_one_local`? I don't understand that method name...

**Comment by thejunglejane** [comment]
I'm...concerned that the tests passed, to be honest.

File: marbles/core/marbles/core/marbles.py (hunk 1 above)
Comment (leifwalsh) [reply marbles/core/marbles/core/marbles.py]: It formats a single local variable. `_format_locals` formats them all, and delegates to this method. Is that clear, should I change the name, add a comment?

Comment (thejunglejane) [reply marbles/core/marbles/core/marbles.py]: Oh I get it. I think making local singular is enough? So we'd have `_format_locals` and `_format_local`. What do you think of that?

**Comment by leifwalsh** [comment]
I'll add to the changelog after #95 goes in, to avoid merge conflicts.

File: marbles/core/tests/test_marbles.py [hunk 2]
```diff
@@ -480,6 +493,14 @@ def test_verify_annotation_locals(self):
         e = ar.exception
         self.assertEqual(e.note.strip(), "some note about 'bar'")
 
+    def test_verify_multiline_locals(self):
```
Comment (thejunglejane) [inline marbles/core/tests/test_marbles.py]: I don't love this test method name; how about `test_mulitline_locals_indentation`?

File: marbles/core/marbles/core/marbles.py [hunk 3]
```diff
@@ -328,9 +328,19 @@ def formattedMsg(self):  # mimic unittest's name for standardMsg
...
+        if '\n' in value_str:
+            value_str = textwrap.indent(value_str, '\t\t')
+            return '\t{0} =\n{1}'.format(name, value_str)
+        else:
+            return '\t{0} = {1}'.format(name, value_str)
+
     @classmethod
     def _format_locals(cls, locals_):
-        return '\n'.join('\t{0}={1}'.format(k, v) for k, v in locals_.items())
+        return '\n'.join(cls._format_local(k, v)
```
Comment (thejunglejane) [inline marbles/core/marbles/core/marbles.py]: Can this go back on one line?

File: docs/changelog.rst [hunk 4]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`92` Format multiline locals better
```
Comment (thejunglejane) [inline docs/changelog.rst]: "Improve indentation of multiline locals"

**Pull request merged**